tools/c7n_azure/tests/templates/keyvault-no-policies.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.KeyVault/vaults",
+            "name": "[concat('cckeyvault2', uniqueString(resourceGroup().id))]",
+            "apiVersion": "2016-10-01",
+            "location": "southcentralus",
+            "tags": {},
+            "scale": null,
+            "properties": {
+                "sku": {
+                    "family": "A",
+                    "name": "Standard"
+                },
+                "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
```

Comment:
This is true for both KV templates: can we get tenantId from az login session somehow instead of hardcoding?

Reply:
You can do smth like this in provision.sh and then reference AZURE_TENANT_ID variable.

`export AZURE_TENANT_ID=$(az account show --query tenantId)`

Reply:
Done